lib/pages/about.dart

```diff
@@ -102,6 +102,19 @@ class About extends StatelessWidget {
                       35),
                 ),
               ),
+              Container(
+                  margin: EdgeInsets.symmetric(horizontal: 20, vertical: 20),
+                  child: Align(
+                    alignment: Alignment.centerLeft,
+                    child: Directionality(
+                        textDirection: TextDirection.ltr,
+                        child: Text(
+                          "Mazilon App Version : 1.1.0",
```

Comment:
why not getting the version from the centralized place. this can be done with https://pub.dev/packages/package_info_plus